### DISCUSSION TITLE
json examples or tutorial

### DISCUSSION THREAD

**Discussion by slamethendry** [janet-lang/janet#1316] [Q&A] [answered]
Hello. I cannot find any docs nor examples / tutorials in Janetdocs.com nor anywhere in Janet's GitHub repo. Any pointer would be appreciated. Thank you.

**Comment by sogaiu** [comment, marked as answer]
May be parts of the [spork](https://github.com/janet-lang/spork) library might be helpful?

There is some usage of json-related bits in [this test file](https://github.com/janet-lang/spork/blob/master/test/suite-json.janet), FWIW.

Also below are some example real-world usages in @CFiggers' janet-lsp fork:

* [success-response](https://github.com/CFiggers/janet-lsp/blob/5da97e0fad5d163871fde32559c118eb1b64bc86/src/rpc.janet)
* [read-message](https://github.com/CFiggers/janet-lsp/blob/5da97e0fad5d163871fde32559c118eb1b64bc86/src/main.janet#L166)

**Comment by slamethendry** [comment]
spork did not show much and I saw that test file, which, I could not understand. The example from janet-lsp fork is more relatable.  Thank you for this.
